todo_app/data/mongo.py

```diff
@@ -0,0 +1,46 @@
+from asyncio.windows_events import NULL
+import requests
+import os
+import json
+from todo_app.data.item import Item
+from oauthlib.oauth2 import WebApplicationClient
+import pymongo
+from pymongo import MongoClient
+import certifi 
+import ssl
+
+
+class Mongo():
+    mongotodos = NULL
+    todos_collection = NULL
+    # Initializing the class, accepting Trello key and token as arguments.
+    def __init__(self):  
+      #  mongo_client = pymongo.MongoClient("${{ secrets.MONGO_URI }}", ssl_cert_reqs=ssl.CERT_NONE)
+       #mongo_client = pymongo.MongoClient(os.getenv("MONGO_URI"), ssl=True, ssl_cert_reqs=ssl.CERT_NONE)   
+        mongo_client = pymongo.MongoClient(os.getenv("MONGO_URI"), tlsAllowInvalidCertificates=True)   
+        self.todos_collection = mongo_client.client.db.todos
+        self.mongotodos = list(self.todos_collection.find())
```

Comment:
I don't think you use this value any more (since we moved to refreshing it each time through the `todos_collection` since otherwise it wouldn't update) so we can get rid of this

Reply:
deleted

Reply:
readded for mockmongo tests